itest/lnd_route_blinding_test.go

```diff
@@ -1461,3 +1461,105 @@ func testBlindedPaymentHTLCReForward(ht *lntest.HarnessTest) {
 		require.Fail(ht, "timeout waiting for sending payment")
 	}
 }
+
+// testGivenBlindedRouteInvoices tests lnd's ability to:
+//   - Assert the error when attempting to create a blinded payment with an
+//     invalid path.
+//   - Create a blinded payment path when the blinded path is partially
+//     pre-specified.
+func testRestrictedBlindedRouteInvoices(ht *lntest.HarnessTest) {
+	// Create a five hop network: Alice -> Bob -> Carol -> Dave -> Erin.
+	chanAmt := btcutil.Amount(100000)
+	cfgs := [][]string{nil, nil, nil, nil, nil}
+	chanPoints, nodes := ht.CreateSimpleNetwork(
```

Comment:
afaict, we dont need to manually close these anymore. I think the framework now handles this for you (see other call-sites of CreateSimpleNetwork)